config/datatypes_conf.xml.sample

```diff
@@ -51,7 +51,7 @@
       <converter file="bed_gff_or_vcf_to_bigwig_converter.xml" target_datatype="bigwig"/>
     </datatype>
     <!-- MSI added Datatypes -->
-    <datatype extension="csv" type="galaxy.datatypes.tabular:Tabular" subclass="True" display_in_upload="true" /> <!-- FIXME: csv is 'tabular'ized data, but not 'tab-delimited'; the class used here is intended for 'tab-delimited' -->
+    <datatype extension="csv" type="galaxy.datatypes.tabular:CSV" subclass="True" display_in_upload="true" />
```

Comment:
This requires discussion because as it is still a subtype of tabular tools will expect from this type to actually be separated by tabs and probably won't work otherwise. It might be viable to leave the old Tabular datatype be and subclass galaxy.datatypes.text into a new datatype instead. 
@blankenberg has more insights, please comment.